packages/core/admin/admin/src/components/RBACProvider/tests/reducer.test.js

```diff
@@ -1,4 +1,4 @@
-import { permissions } from '../../../../../../../admin-test-utils/lib/fixtures';
+import { permissions } from '@strapi/admin-test-utils/lib/fixtures';
```

Comment:
Should `@strapi/admin-test-utils` be listed as a dev-dependency then?

Reply:
eslint didn't complain so I guess it was already the case I'll check